handlers/remove.go

```diff
@@ -48,8 +54,10 @@ func PurgeMachine(event *events.Event, apiClient *client.RancherClient) error {
 		return err
 	}
 
-	log.Printf("Done purging machine. ResourceId: %v. ExternalId: %v.", event.ResourceId,
-		machine.ExternalId)
+	log.WithFields(log.Fields{
```

Comment:
Can delete this log statement all together. We have the log statement on the entry to this function.